docs/Antora/modules/getting_started/pages/quick_start_application.adoc

```diff
@@ -559,6 +538,75 @@ for overridden_type_value in reader.read(10):
 ----
 ====
 
+Reding signal last value can be simplified by using signal.last_value method. The following example shows how to read the last value of the signal.
```

Comment:
Should we use a specific language syntax outside the code examples or just "by using the signal's last value method"?